src/components/shared/Modal.js

```diff
@@ -20,6 +20,8 @@ type ModalProps = {
   handleClose: () => any
 };
 
+export const transitionTimeout = 175;
```

Comment:
Just curious, why should it be exactly 175?

Reply:
That was the value that the `timeout` prop of `<Transition/>` already had. I just made it exportable. I'm not sure why we have a delay for the transition of modals to be honest. :|

Reply:
https://reactcommunity.org/react-transition-group/#Transition-prop-timeout

> The duration of the transition, in milliseconds. Required unless addEventListener is provided.

Reply:
Huh, weird. I thought that the timeout was just a delay and when it ends, the transition is fired. I gave it a value of 2000ms just to check what it does and the transition speed of the modal appearance is the same. So maybe it is a delay. Am I missing something? 🤔 

I think that we can make it faster and give the modal appearance a more responsive feel if we make the timeout 25ms.